test/extensions/access_loggers/wasm/test_data/BUILD

```diff
@@ -26,7 +26,7 @@ envoy_cc_library(
     ],
 )
 
-wasm_cc_binary(
+envoy_wasm_cc_binary(
```

Comment:
This could have stayed `wasm_cc_binary` with `repository = "@envoy"`, instead of creating a new rule, IMHO.

Reply:
Then I would have to add that to every target.   This is simpler IMO.

Reply:
Correct, and I agree, but we do the same (i.e. add `repository = "@envoy"` to every target) for `envoy_cc_library`, etc., so this way it would be at least consistent.

Reply:
That is the great thing about standards, there are so many to choose to be consistent with.  I choose the envoy_cc_library standard which adds custom envoy logic into a wrapper function rather than the repository="@envoy" standard.  I think the latter is a bug and should be fixed :)

Reply:
So you're choosing to be consistently inconsistent? :)

(I approved this PR when making the initial comment, so feel free to merge it as-is, I don't really care)